.devcontainer/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM --platform=$BUILDPLATFORM ubuntu:24.04
+ARG TARGETARCH
+
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get update && \
+    apt-get install -y \
+    adwaita-qt \
+    alsa-utils \
+    build-essential \
+    clangd \
+    gdb \
+    git \
+    libadwaitaqt1 \
+    pipewire \
+    pipewire-alsa \
+    pulseaudio-utils \
+    python3-pip \
+    qt6-wayland \
+    sudo \
```

Comment:
Do we need sudo? For my understanding you are already superuser in a devcontainer. 
I am afraid there are some more command line tools missing. Isn't there not a different base container of a devcontainer feature that comes with all this?